### ISSUE TITLE
Provide popularity-related information

### ISSUE THREAD

**Issue by timhoffm** [matplotlib/mpl-third-party#116]
Provide popularity-related information for the projects, e.g.

- github stars ![GitHub Repo stars](https://img.shields.io/github/stars/matplotlib/matplotlib?style=social)
- PyPI installs ![PyPI - Downloads](https://img.shields.io/pypi/dm/matplotlib)

Format and content t.b.d.

This should help users to see whether the project is something substantial or rather a small side-project.

**Comment by jklymak** [comment]
We had this originally, and its relatively easy to add, but the table formatting was hard (for me).

**Comment by timhoffm** [comment]
Yes, the available width is somewhat small for additional information. I suggest whether we can remove the (unused) side bars and use that space as well.
